pss/pss_test.go

```diff
@@ -0,0 +1,73 @@
+package pss
+
+import (
+	"context"
+	"crypto/rand"
+	"io/ioutil"
+	"os"
+	"reflect"
+	"testing"
+
+	"github.com/ethersphere/swarm/chunk"
+	trojan "github.com/ethersphere/swarm/pss/trojan"
+	"github.com/ethersphere/swarm/storage/localstore"
+)
+
+func TestTrojanChunkRetrieval(t *testing.T) {
+	var err error
+	ctx := context.TODO()
+
+	localStore := newMockLocalStore(t)
+
+	testTargets := [][]byte{
+		{57, 120},
+		{209, 156},
+		{156, 38},
+		{89, 19},
+		{22, 129}}
+	payload := []byte("RECOVERY CHUNK")
+	topic := trojan.NewTopic("RECOVERY")
+
+	// call Send to store trojanChunk in store
```

Comment:
move this comment to line `35`, and call it `trojan chunk` as `trojanChunk` is no longer a type

Reply:
fixed